src/qt/pivx/sendcustomfeedialog.cpp

```diff
@@ -74,8 +78,9 @@ void SendCustomFeeDialog::onCustomChecked(){
     ui->checkBoxRecommended->setChecked(!isChecked);
 
     if(walletModel && ui->lineEditCustomFee->text().isEmpty()) {
-        feeRate = CWallet::minTxFee;
-        ui->lineEditCustomFee->setText(BitcoinUnits::format(walletModel->getOptionsModel()->getDisplayUnit(), feeRate.GetFeePerK()));
+        CAmount nFee;
```

Comment:
Remember that zero fee txs are not accepted in the mempool, this should be initialized to the minimum fee.

Reply:
:+1:

Reply:
In the `optionModel` for when `pwalletMain` is null, but also directly in `CWallet::SetNull()` for `nCustomFee` initialization. Fixed.